apps/launcher_app.lua

```diff
@@ -55,12 +55,12 @@ laptop.register_app("launcher", {
 		end
 		if app.app_info then
 			if #mtos.sysram.stack > 0 then
-				formspec = formspec.."label[0.8,-0.29;"..app.app_info.."]"
+				formspec = formspec..mtos.theme:get_label("0.8,-0.29", app.app_info, "titlebar")
 			else
-				formspec = formspec.."label[-0.1,-0.29;"..app.app_info.."]"
+				formspec = formspec..mtos.theme:get_label("0.8,-0.29", app.app_info, "titlebar")
```

Comment:
The old position value was "-0.1,-0.29". Te titlebar text is more at the left site if no back button visible